core/tests/build_sources/assets/constants.js

```diff
@@ -8,8 +8,9 @@
  *    functions.
  */
 var constants = {
-  // Whether to allow custom event reporting to Google Analytics.
-  "CAN_SEND_ANALYTICS_EVENTS": false,
+  // Analytics Sandbox mode allows using mock gtag function call instead of
+  // invoking actual gtag.
+  "ANALYTICS_SANDBOX_MODE": true,
```

Comment:
Why is this still changed?

(If unintentional -- in general, in the future, I suggest doing a self-review immediately before/after you push changes to try and catch things like this.)

Reply:
Sure, will take care in the future.